docs/TPC-H.md

```diff
@@ -30,6 +31,33 @@ Change the bucket name if it already exists.
 
 During data loading, Pixels will automatically create the folders in the bucket to store the files in each table.
 
+## Install MinIO*
+
+MinIO is an alternative to the AWS S3 storage. 
+It provides the similar functionality as AWS S3 storage and supports [single node deployment](https://min.io/docs/minio/linux/index.html) as well as [multiple nodes deployment](https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html).
+
+Follow the documentation, you can establish the installation. 
+After login to the administrative website, create your own access key and secret key for `pixels-tpch` and `pixels-lambda-test` (defined as `executor.intermediate.folder` and `executor.output.folder` in `pixels.properties`) buckets.
+You can follow the read-write policy to give this access key the read-write policy to `pixels-tpch` and `pixels-lambda-test` bucket as follows:
+
+```properties
+{
+ "Version": "2012-10-17",
+ "Statement": [
+  {
```

Comment:
As Minio is fast evolving, we'd better not give the installation instructions of minio in our docs.